README.md

```diff
@@ -54,6 +54,18 @@ The boilerplate includes
 - [Prettier](https://github.com/prettier/prettier)
 - [Reactotron](https://github.com/infinitered/reactotron)
 
+### Bootstrap Kick off
+
+For bootstraping your own project using the template generator script you'll need to follow these steps:
+
+1. run `yo yourPathToWolmo/generators/app/index.js -v` you'll need [Yeoman](https://yeoman.io/learning/index.html) installed for this.
```

Comment:
I'd remove the `-v` arg and add a point clarifying that we can see the logger using `-v`